registry/example/registry.json

```diff
@@ -2,14 +2,41 @@
   {
     "name": "ODO",
     "description": "OpenShift Developer CLI",
-    "version": "v1",
-    "homepage": "https://www.example.com",
+    "version": "v2.0.7",
+    "homepage": "https://developers.redhat.com/products/odo/overview",
     "binaries": {
       "darwin-amd64": "https://mirror.openshift.com/pub/openshift-v4/clients/odo/v2.0.7/odo-darwin-amd64",
       "linux-arm64": "https://mirror.openshift.com/pub/openshift-v4/clients/odo/v2.0.7/odo-linux-arm64"
     },
     "dependencies": {},
     "tags": [],
     "license": "MIT"
+  },
+  {
+    "name": "OC",
+    "description": "OpenShift CLI",
+    "version": "v4.6",
+    "homepage": "https://docs.openshift.com/container-platform/4.6/cli_reference/openshift_cli/getting-started-cli.html",
+    "binaries": {
+      "linux": "https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.6/linux/oc.tar.gz",
```

Comment:
Not sure if we want to support zipped binaries.

Reply:
I tried to get an uncompressed binary but couldn't find any.

Even the documentation states that first we need to download this zipped binary and then unzip it while installation.
(Ref: https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli-on-linux_cli-developer-commands)

Should I add any other CLI to the registry in place of this one?

Reply:
I would leave it. Great corner case